elixir/deploy

```diff
@@ -4,10 +4,28 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
+REPO=${REPO:-https://packages.erlang-solutions.com}
```

Comment:
Maybe use a more descriptive name than simply `REPO` (`ERLANG_REPO`?) would be better to avoid conflicts or accidents with apps setting the env `REPO` for other purposes.

Reply:
great! Ill change it.